core/domain/stats_jobs_one_off_test.py

```diff
@@ -2020,3 +2020,133 @@ def test_statistics_customization_args_audit(self):
+        Assumes the existence of a class constant ONE_OFF_JOB_CLASS, pointing
+        to the queue under test.
+
+        Returns:
+            *. The output of the one off job.
+        """
+        job_id = self.ONE_OFF_JOB_CLASS.create_new()
+        self.assertEqual(self.count_one_off_jobs_in_queue(), 0)
+        self.ONE_OFF_JOB_CLASS.enqueue(job_id)
+        self.assertEqual(self.count_one_off_jobs_in_queue(), 1)
+        self.process_and_flush_pending_mapreduce_tasks()
+        self.assertEqual(self.count_one_off_jobs_in_queue(), 0)
+        return [
+            ast.literal_eval(output)
+            for output in self.ONE_OFF_JOB_CLASS.get_output(job_id)
+        ]
+
+    def get_exp_issues(self, exp_version=1):
+        """Fetches the ExplorationIssuesModel for the given version."""
+        return stats_models.ExplorationIssuesModel.get_model(
+            self.EXP_ID, exp_version)
+
+    def append_exp_issue(self, exp_issues_model, playthrough_ids):
+        """Appends a new ExplorationIssue to the given model.
+
+        Args:
+            exp_issues_model: ExplorationIssuesModel. The model to append a new
+                issue to.
+            playthrough_ids: list(str). The playthrough ids to include.
+        """
+        issue_type = 'EarlyQuit'
+        issue_customization_args = {
+            'state_name': {'value': ''},
+            'time_spent_in_exp_in_msecs': {'value': 0},
+        }
+        schema_version = 1
+        is_valid = True
+        exp_issues_model.unresolved_issues.append(
+            stats_domain.ExplorationIssue(
+                issue_type, issue_customization_args, playthrough_ids,
+                schema_version, is_valid).to_dict())
+        exp_issues_model.update_timestamps()
+        exp_issues_model.put()
+
+    def create_playthrough_model(self, exp_version=1):
+        """Creates a new playthrough model and returns its ID.
+
+        Args:
+            exp_version: int. The exploration version the playthrough was
+                recorded within.
+
+        Returns:
+            str. The ID of the newly created playthrough.
+        """
+        return stats_models.PlaythroughModel.create(
+            exp_id=self.EXP_ID,
+            exp_version=exp_version,
+            issue_type='EarlyQuit',
+            issue_customization_args={
+                'state_name': {'value': ''},
+                'time_spent_in_exp_in_msecs': {'value': 0},
+            },
+            actions=[{
+                'action_type': 'ExplorationStart',
+                'action_customization_args': {'state_name': {'value': ''}},
+                'schema_version': 1,
+            }]
+        )
+
+    def test_when_nothing_exists(self):
+        self.assertEqual(self.run_one_off_job(), [])
+
+    def test_when_everything_is_ok(self):
+        self.save_new_valid_exploration(self.EXP_ID, self.OWNER_ID)
+
+        self.assertItemsEqual(self.run_one_off_job(), [
+            ['Existing ExplorationIssuesModel(s) reset', 1],
```

Comment:
This feels slightly misleading--nothing was actually reset. Should anything be outputted for this case?

Reply:
It was reset (the timestamp still changed), it just so happened to be for an already-empty set of issues. I've renamed it to wipe since reset is probably a bad name, but I still think it should stay.

And yes, we are required to report successful cases in aggregate: 
![image](https://user-images.githubusercontent.com/5094060/101232307-f76fa380-367e-11eb-9801-694787211629.png)

Reply:
Thanks @brianrodri. I interpreted the code originally to be doing nothing in this case ('nothing' isn't the same as success/failure), but given that there _is_ a side effect then outputting that makes sense.